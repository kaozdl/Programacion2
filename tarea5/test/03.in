# Caso 03
ingresar_localidades (99,Guaviyu) (80,Minas) (60,TreintayTres) (21,Flores) (1,PuntaEspinillo) .
iniciar_pedidos 10 .
ingresar_pedidos Guaviyu (1000,chapas) 4 Minas (500,ropa) 10 TreintayTres (2000,comestibles) 3 .
ingresar_pedidos Flores (900,chapas) 10 Minas (5000,comestibles) 2 .
procesar_rango 1
procesar_rango 2
procesar_rango 3
procesar_rango 4
procesar_rango 5
imprimir_totales
Fin

