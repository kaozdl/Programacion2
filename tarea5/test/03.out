1>2>3>
 IDENTIFICADORES DE LOCALIDADES 
(1,Flores)(2,Guaviyu)(3,Minas)(4,PuntaEspinillo)(5,TreintayTres)
4>5>6>PuntaEspinillo

7>Flores
(900,chapas)
8>TreintayTres
(2000,comestibles)
9>Minas
(5000,comestibles)(500,ropa)
10>Guaviyu
(1000,chapas)
11>INFORME POR RANGOS
0
0
0
0
0
12>Fin.
